htdocs/don/payment/payment.php

```diff
@@ -192,14 +192,14 @@
 	print '</tr>';
 
 	print '<tr><td class="fieldrequired">'.$langs->trans("PaymentMode").'</td><td colspan="2">';
-	$form->select_types_paiements(GETPOSTISSET("paymenttype") ? GETPOST("paymenttype") : $object->paymenttype, "paymenttype");
+	$form->select_types_paiements(GETPOSTISSET("paymenttype") ? GETPOST("paymenttype") : $object->fk_typepayment, "paymenttype");
```

Comment:
There was a confusion (and there is still a confusion) between paymenttype and fk_typepayment that are 2 variables used for same thing when fk_typepayment should contain the ID (rowid) of type of payment and paymenttype should contains the code (but still contains an ID sometime).
So this change is sensitive due to old code not correctly written.
So to avoid regression, can you tell me if you succeed in creating a payment on a donation after making this change ?